src/main/asciidoc/index.adoc

```diff
@@ -268,6 +269,21 @@ If you want to deploy a verticle as a worker verticle you do that with {@link io
 Worker verticle instances are never executed concurrently by Vert.x by more than one thread, but can executed by
 different threads at different times.
 
+=== Virtual thread verticles
+
+A virtual thread verticle is just like a standard verticle but it's executed using a virtual threads, rather than using an event loop.
```

Comment:
```suggestion
A virtual thread verticle is just like a standard verticle but it's executed using virtual threads, rather than using an event loop.
```